sensor_msgs/Image image
---
int32 resx
int32 resy
int32 label

